README.adoc

```diff
@@ -1,5 +1,14 @@
 = Neo4j Cypher Manual
 
+This page covers the following topics:
+
+* xref:README.adoc#build-locally[]
+* xref:README.adoc#raise-prs[]
+* xref:README.adoc#documenting-changes[]
+* xref:README.adoc#cypher-gql[]
+
+
+[[build-locally]]
```

Comment:
not sure these have to be set
asciidoc generates anchors for headings by default - it should be `#building-locally` for the line below

edit: just noticed, on github, both anchors are present now (eg https://github.com/JPryce-Aklundh/docs-cypher/tree/updated_readme?tab=readme-ov-file#documenting-changes and https://github.com/JPryce-Aklundh/docs-cypher/tree/updated_readme?tab=readme-ov-file#documenting-changes-to-cypher)